src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocContentLocationCheck.java

```diff
@@ -208,12 +208,12 @@ public void visitToken(DetailAST ast) {
             final String commentContent = JavadocUtil.getJavadocCommentContent(ast);
             final int indexOfFirstNonBlankLine = findIndexOfFirstNonBlankLine(commentContent);
             if (indexOfFirstNonBlankLine >= 0) {
-                if (location == JavadocContentLocationOption.FIRST_LINE) {
-                    if (indexOfFirstNonBlankLine != 0) {
-                        log(ast, MSG_JAVADOC_CONTENT_FIRST_LINE);
-                    }
+                if (location == JavadocContentLocationOption.FIRST_LINE
+                        && indexOfFirstNonBlankLine != 0) {
+                    log(ast, MSG_JAVADOC_CONTENT_FIRST_LINE);
                 }
-                else if (indexOfFirstNonBlankLine != 1) {
+                if (location == JavadocContentLocationOption.SECOND_LINE
```

Comment:
Can we make it `else if`? If the first `if` is true, no reason to check the second one.

Reply:
Done